src/components/AppHeader.vue

```diff
@@ -89,25 +108,29 @@ export default defineComponent({
 	},
 });
 </script>
-<style scoped>
-.header {
-	width: 1400px;
-	height: auto;
-	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
-	font-size: 15px !important;
-	padding: 10px;
-	margin: auto;
+<style>
+#components-page-header-demo-responsive .content {
+	display: flex;
+}
+.ant-page-header-heading-extra {
+	align-self: center;
 }
-.veritcal_center {
-	align-items: center;
+.ant-page-header-heading-left {
+	flex-shrink: 0;
 }
-.flexbox {
-	display: flex;
+.dao-table {
+	max-width: 1000px;
+	margin: auto;
+}
+.menu {
+	text-align: center;
+	margin-bottom: 1rem;
 }
 .menu_option {
-	font-size: 20px;
+	font-size: 1.8rem !important;
```

Comment:
let's use LESS or SCSS variable. We should avoid creating ad hock custom font size. Instead create varibles, like: 
```
f-size1  = 1.2rem, 
f-size2  = ....
```